testing.py

```diff
@@ -387,11 +388,11 @@ def _lane(self, display_name=None, library=None,
     ):
         display_name = display_name or self._str
         library = library or self._default_library
-        lane, is_new = get_one_or_create(
+        lane, is_new = create(
```

Comment:
Using this method to create two lanes with different genres gave you one lane with two genres.